### DISCUSSION TITLE
【2022年上半期】Wiki整備計画（施設情報の拡充）

### DISCUSSION THREAD

**Discussion by A-B-S-I** [freeserverproject/freeserver-wiki#111] [Ideas]
- Wiki編集ガイドライン、鯖民向けのを周知させる、 ~~フォーマット一覧をそこにぶら下げる。~~ 各ページ例を作成してガイドラインのページから飛べるようにする。

- メインワールド直下に、ランドマークベースの地域①を列挙する。 その下の施設情報やら中身は、鯖民におまかせ。 

- 施設情報②を別でディレクトリを作っておいて、そこにも列挙。

→施設ごとに記事を独立させて、①②どちらからもアクセスできるようにする。

→地域①の情報は、後から募集できるようにする。 マップのピンはスパベースが整備できたら。

**Comment by graycat27** [comment]
> メインワールド直下に

メインのネザーとエンドをお忘れなく。

地域毎一覧と、施設カテゴリ毎(?)をそれぞれ作成して、それぞれから各施設記事へジャンプできるようにするのは良いアイデア。地域の分け方とかのセンスはまぁ、がんばって()

**Comment by graycat27** [comment]
カッコイイissueタイトルにすると、「【2022年上半期】Wiki整備計画（施設情報の拡充）」とかじゃないですかね（小声）

**Comment by Nakatai-0322** [comment]
>  Wiki編集ガイドライン、鯖民向けのを周知させる、フォーマット一覧をそこにぶら下げる。

既存のguideline.mdにぶら下げる方針だと、guideline.mdが肥大化しちゃいそうですね...

**Comment by A-B-S-I** [comment]
フォーマット自体は別記事のつもりでした、ページ丸ごとの例示になるでしょうし？修正しまふ

**Comment by Nakatai-0322** [comment]
了解です。

**Comment by Nakatai-0322** [comment]
Wiki編集ガイドライン（鯖民用）近いうちに書きますね。（一応報告）

**Comment by Nakatai-0322** [comment]
https://github.com/freeserverproject/freeserver-wiki/tree/add-sabamin-wiki-guideline
に書いてます。

**Comment by A-B-S-I** [comment]
guideline.mdとhow-to-edit.mdのことですよね？
21日までに中身見ますね〜

**Comment by A-B-S-I** [comment]
鯖民の方々に、Wiki編集のキャンペーンでも仕掛けようかと思っているのですが、いかがいたしやしょ。
あと作り終わってないのってなんです…？

**Reply by Nakatai-0322** [reply]
現状作り終わってないのは...

各地域の紹介的なやつとかですかね。

**Reply by A-B-S-I** [reply]
私が見る限り、ページの作り方の、各フォーマットの例示されたページとかもないのかな？

**Reply by A-B-S-I** [reply]
ディレクトリだけもりもり今作ってます、何か提案修正等あればご指摘ください

**Reply by A-B-S-I** [reply]
ゲーム内で公共性の高い、農場、牧場、トラップ等を想定して、public-facilitiesを作ってみました。
 観光施設という括りになる場合は、地域のページがあるディレクトリにぶら下げる形を想定しています。
